lang/src/main/java/org/eclipse/steady/goals/GoalClient.java

```diff
@@ -16,31 +16,37 @@
  * SPDX-License-Identifier: Apache-2.0
  * SPDX-FileCopyrightText: Copyright (c) 2018-2020 SAP SE or an SAP affiliate company and Eclipse Steady contributors
  */
-package org.eclipse.steady.kb.exception;
+package org.eclipse.steady.shared.enums;
```

Comment:
This class has been moved from module `shared` to module `kb-importer`. The Java file in the latter is now empty. Why is that needed?